code/game/turfs/flooring/flooring.dm

```diff
@@ -140,6 +140,13 @@ var/list/flooring_types
 		'sound/effects/footstep/grass3.ogg',
 		'sound/effects/footstep/grass4.ogg'))
 
+/decl/flooring/grass/outdoors // VOREStation Edit
+	flags = TURF_REMOVE_SHOVEL // VOREStation Edit
+
+/decl/flooring/grass/outdoors/forest // VOREStation Edit
+	icon = 'icons/turf/outdoors.dmi' // VOREStation Edit
+	icon_base = "grass-dark" // VOREStation Edit
```

Comment:
Make a flooring_vr.dm and put these in there. This makes it not need all those //VOREStation Edits. Make sure to check the file in the .dme as well.

Reply:
> Make a flooring_vr.dm and put these in there. This makes it not need all those //VOREStation Edits. Make sure to check the file in the .dme as well.

There's already a flooring_vr so I put it in there